src/main/scala/com/fulcrumgenomics/umi/CollectDuplexSeqMetrics.scala

```diff
@@ -340,26 +376,48 @@ class CollectDuplexSeqMetrics
     }
   }
 
-  /** Updates all the various metrics for the UMIs contained on the given records. */
-  private def updateUmiMetrics(ssGroups: Seq[Seq[SamRecord]]): Unit = {
+  /**
+    * Updates all the various metrics for the UMIs contained on the given records.
+    *
+    * @param ssGroups a Seq of length one or two containing the one or two single stranded tag families
+    *                 that comprise a single double-stranded tag family.  If there are two single-stranded
```

Comment:
can you change `single` to `singular`; I got confused as to this mythical "single-double" tag family.